src/components/calcite-tab-nav/calcite-tab-nav.tsx

```diff
@@ -199,18 +266,63 @@ export class CalciteTabNav {
    */
   @Event() calciteTabChange!: EventEmitter<TabChangeEventDetail>;
 
+  //--------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  //--------------------------------------------------------------------------
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private State/Props
+  //
+  //--------------------------------------------------------------------------
+
+  private tabNavEl: HTMLElement;
```

Comment:
question: do we know the element types here or could it be any HTMLElement?

Reply:
I think we could type this more specifically to the `HTMLCalciteTabNavElement`, will try it out